modules/cluster-logging-collector-rsyslog.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-rsyslog_{context}"]
+= Configuring the Fluentd remose-syslog plug-in to send logs to an external log aggregator
+
+You can configure Fluentd to send a copy of its logs to an external Rsyslog server, 
+and not the default Elasticsearch, using the *fluent-plugin-remote-syslog* plug-in.
+
+[NOTE]
+====
+This legacy *fluent-plugin-remote-syslog* method is deprecated and will be removed in a future release.
+====
+
+There are two versions of the *fluent-plugin-remote-syslog* plug-in:
+
+* *out_syslog*. Use `syslog` in the ConfigMap for the non-buffered implementation, communicating through UDP. 	
+This implementation is insecure, and should only be used in environments where you can guarantee no snooping on the connection.
+* *out_syslog_buffered*: Use `syslog_buffered` in the ConfigMap for the buffered implementation, communicating through TCP.
+
+To configure the *fluent-plugin-remote-syslog* plug-in, create a ConfigMap called `secure-forward` in the `openshift-logging` namespace. 
+
+[IMPORTANT]
+====
+Changes introduced by the new log forward feature modified the support for *fluent-plugin-remote-syslog* starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure *fluent-plugin-remote-syslog*. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed. Before upgrading cluster logging, you can copy your current `secure-forward.conf` section and use the copied data when you create the `secure-forward` ConfigMap. 
+
+Additionally, you can add any certificates required by your configuration to a secret named `secure-forward` that will be mounted to the Fluentd Pods.
+====
+
+You can use multiple remote syslog servers by specifying separate `<store>` stanzas in the configuration file.
+
+.Sample `secure-forward.conf`
```

Comment:
I don't think this should be secure-forward, since that implies the old *secure_forward* protocol which is different